src/redux/actions/taskflows.js

```diff
@@ -381,6 +391,7 @@ client.onEvent((resp) => {
       }
       case 'cluster': {
         // fetch clusters
+        updateTaskflowMeta();
         dispatch(clusterActions.fetchClusters(null));
```

Comment:
Yes, I just realized too there's another method named `updateTaskflowMetadata` in the same file so I'l be renaming it too.